R/data.table.R

```diff
@@ -284,7 +284,7 @@ replace_dot_alias = function(e) {
     } else if (is.name(jsub)) {
       if (startsWith(as.character(jsub), "..")) stop("Internal error:  DT[, ..var] should be dealt with by the branch above now.") # nocov
       if (!with && !exists(as.character(jsub), where=parent.frame()))
-        stopf("Variable '%s' is not found in calling scope. Looking in calling scope because you set with=FALSE. Also, please use .. symbol prefix and remove with=FALSE.", jsub)
+        stopf("Variable '%s' is not found in calling scope. Looking in calling scope because you set with=FALSE. Also, please use .. symbol prefix and remove with=FALSE.", as.character(jsub))
```

Comment:
Couldn't this new helper do automatically as.character?

Reply:
the issue is `sprintf("%s", as.name('a'))` vs `paste(as.name('a'))`. I suppose we could do something like `vapply(list(...), function(x) if (is.language(x)) as.character(x) else x)` but seems like overkill -- this is the only time in the code base where we need the conversion.